### PR TITLE
Adding an icon indicating lost connection to sc monitoring

### PR DESCRIPTION
**Pull request by WojcikMike**
@sergioc I finished it from my end. The icons are added. I marked it as WIp to allow you to do what you need to do

### CODE REVIEW

File: src/ServicePulse.Host/app/img/monitoring-lost.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/img/monitoring-lost.svg]: This file needs to be added to the csproj file. I think the release build might break unless this is included in the csproj

File: src/ServicePulse.Host/app/js/app.constants.js [hunk 2]
```diff
@@ -5,8 +5,8 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             default_route: '/dashboard',
-            service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            service_control_url: 'http://localhost:33533/api/',
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/app.constants.js]: Reason for the change?

File: src/ServicePulse.Host/app/css/particular.css [hunk 3]
```diff
@@ -1985,4 +1999,10 @@ i.fa-exclamation-triangle {
     text-transform: uppercase;
     margin-bottom: 6px;
     font-size: 13px;
+}
+
+.pa-monitoring-lost {
+    background-image: url('../img/monitoring-lost.svg');
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/css/particular.css]: As per our debug session, we'll need 2 rules here depending on the path

File: src/ServicePulse.Host/app/js/app.constants.js (hunk 2 above)
Comment (sergioc) [reply src/ServicePulse.Host/app/js/app.constants.js]: No reason in particular, will revert when not WIP anymore.

**Comment by WojcikMike** [comment]
Ready to be reviewed and merged

**Comment by sergioc** [comment]
Latest version is 1.11.0, so make milestone 1.12.0?

**Pull request merged**